cmsl1t/playground/eventreader.py

```diff
@@ -31,12 +31,14 @@
 }
 
 
-def get_trees(load_emu_trees, load_reco_trees):
+def get_trees(load_emu_trees, load_reco_trees, load_vertex_trees):
```

Comment:
If you are changing the behaviour of crucial classes such as eventreader, please update the tests:
https://github.com/cms-l1t-offline/cms-l1t-analysis/blob/master/test/playground/test_eventreader.py#L11

Alternatively, you can set a sensible default for the new parameter.

Reply:
Added load_vertex and extra test to test_eventreader.py